erpnext/templates/pages/cart.html

```diff
@@ -54,6 +49,20 @@
 	{% else %}
 	<p class="text-muted">{{ _('Your cart is Empty') }}</p>
 	{% endif %}
+	{% if doc.items and cart_settings.enable_checkout %}
+	<div class="place-order-container">
+		<button class="btn btn-primary btn-place-order" type="button">
+			{{ _("Place Order") }}
+		</button>
+	</div>
+	{% endif %}
+	{% if doc.items and not cart_settings.enable_checkout %}
+	<div class="place-order-container"></div>
```

Comment:
You've accidentally closed the `div` here, this won't display properly

Nest the conditions to make it easier to read, something like (pseudo-code):

```
if doc.items:
	<div>
		if cart_settings.enable_checkout:
			<button>Place Order</button>
		else:
			<button>Request for Quotation</button>
	</div>
```